packages/eslint-config/shared.js

```diff
@@ -81,20 +106,21 @@ module.exports = {
             position: 'after',
           },
           {
-            pattern: 'src/lib/test',
-            group: 'parent',
-            position: 'before',
-          },
-          {
-            pattern: 'src/**',
-            group: 'parent',
+            // Matches src/directives/**/*.{js,ts}, src/services/**/*.{js,ts}, src/graphql/**/*.sdl.{js,ts}
+            pattern: 'src/*/\\*\\*/\\*\\.[^/]*\\{js,ts\\}$',
```

Comment:
Looks like this needs another tweak; it doesn't seem to be matching the above examples anymore

Reply:
For full context, this string is fed to [minimatch](https://github.com/isaacs/minimatch) which turns it into a RegExp. The way I test if it works is by:

```js
minimatch('src/directives/**/*.{js,ts}', 'src/*/\\*\\*/\\*\\.[^/]*\\{js,ts\\}$')
```

Reply:
Running [makeRe](https://github.com/isaacs/minimatch#methods), this is what the above expands to:

```js
minimatch.makeRe('src/*/\\*\\*/\\*\\.[^/]*\\{js,ts\\}$')
/^(?:src\/(?!\.)(?=.)[^/]*?\/\*\*\/\*\.\[\^\/(?=.)\][^/]*?\{js,ts\}\$)$/
```

Reply:
I simplified this a bunch in https://github.com/redwoodjs/redwood/pull/6213/commits/dc3d65846535768e3ea07cfb9debf0439a312ad0 and https://github.com/redwoodjs/redwood/pull/6213/commits/d44822695b6dee836787e983bfafea350d81ad8c

Reply:
Thanks for the explanation, and sorry for making you spend time on this